source/connect/connection-pools.txt

```diff
@@ -0,0 +1,106 @@
+.. _pymongo-connection-pools:
+
+================
+Connection Pools
+================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn about how {+driver-short+} uses connection pools to manage
+connections to a MongoDB deployment and how you can configure connection pool settings
+in your application.
+
+A connection pool is a cache of open database connections maintained by {+driver-short+}.
+When your application requests a connection to MongoDB, {+driver-short+} seamlessly
+gets a connection from the pool, performs operations, and returns the connection
+to the pool for reuse.
+
+Connection pools help reduce application latency and the number of times new connections
+are created by {+driver-short+}.
+
+Configuring Connection Pools
+----------------------------
+
+You can specify the following connection pool settings in your ``MongoClient`` object or in
+your connection URI:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Setting
+     - Description
+
+   * - ``connectTimeoutMS``
+     - | Sets the time that {+driver-short+} waits when connecting a new
+         socket before timing out.
+       | Defaults to ``20000``
+
+   * - ``maxConnecting``
+     - | Sets the maximum number of connections that each pool can have at once.
```

Comment:
The maximum number of connections that each pool can _establish concurrently_. If this limit is reached, further requests wait until one of these connections are established or another in use connection is checked back into the pool.